migrate.md

```diff
@@ -0,0 +1,111 @@
+{:swift: .ph data-hd-programlang='swift'}
+
+# Migrating from Cloud Foundry
+{: #migrate}
+
+Migrate a service instance to move it from its current Cloud Foundry org and space to a resource group.
+{: shortdesc}
+
+{{site.data.keyword.cloud_notm}} moved from using Cloud Foundry to using resource groups.
+
+Resource groups offer these benefits over Cloud Foundry:
+
+- Finer-grained access control by using IBM Cloud Identity and Access Management (IAM)
+- Ability to connect service instances to apps and service across different regions
+- Simplifies the capture of usage data per group
+
+If you created service instances before November 2018, then depending on the location in which your instance is hosted, it might be using Cloud Foundry instead of resource groups.
+
+## Migrating a service instance
+{: #migrate-task}
+
+If you want to learn more about the migration process before you begin, see the [IBM Cloud migration documentation ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-migrate){: new_window}.
+
+Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more details, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: new_window}.
+{: note}
+
+To migrate your service instance, complete these steps:
+
+1.  Determine which resource group you want to move the service instance to first.
+
+    See [Best practices for organizing resource in a resource group ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-bp_resourcegroups) for tips.
+
+    With a Premium plan, you can migrate multiple instances. However, the instances must all be migrated to the same resource group.
+
+1.  From the *Cloud Foundry Services* section of the {{site.data.keyword.Bluemix_notm}} resource list, click the migrate icon ![Migrate](images/migrate.svg) for the instance you want to migrate, and then click **Migrate** from the popup.
+
+1.  Click **Continue**, and then choose a resource group.
+
+    If you have not created a resource group, you can create one now. A **default** resource group is available. Take time to understand how the groups are used and create one if necessary. The resource group that you choose here *cannot* be changed later.
+
+1.  Click **Migrate**.
+
+    A message is displayed when the process is done. If you have other service instances to migrate, you can continue migrating other service instances, or click **Done**.
+
+1.  **Premium plan one-time step**: If the service instance you are migrating was created as part of a Premium plan, then you must inform the IBM support team that you are migrating a Premium plan instance. To do so, create a case from [IBM Cloud Support ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://cloud.ibm.com/unifiedsupport/supportcenter){: new_window}.
```

Comment:
Acrolinx will most probably request changing `service instance you are migrating` to `service instance that you are migrating`.  It really like relative pronouns. :-)